### DISCUSSION TITLE
CSS animation not working

### DISCUSSION THREAD

**Discussion by Biki-das** [tailwindlabs/tailwindcss#10173] [Help] [answered]
I have made a  custom animation which works perfectly when i throw it against any HTML element as a class, problem starts when i try to add it to a element using JavaScript, like element.classList.add("animation-name"), it does not work? any idea of what i am doing wrong

**Comment by benborgers** [comment, marked as answer]
Tailwind might not be generating that class because it's not present in your HTML. 

You could add the JavaScript file to the `content` array of your Tailwind config file so it gets scanned, or manually add the class to Tailwind's [safelist](https://tailwindcss.com/docs/content-configuration#safelisting-classes).

**Reply by Biki-das** [reply]
Thanks a lot, @benborgers , your solution really worked!, i added my main.js file to the  tailwind content config and this has made the tailwind class scanned!
